client/state/products-list/test/selectors.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import reducer from '../reducer';
+import {
+	PRODUCTS_LIST_RECEIVE,
+	PRODUCTS_LIST_REQUEST,
+} from 'state/action-types';
+import {
+	getProductsList,
+	isProductsListFetching,
+} from '../selectors';
+import {
+	guided_transfer,
+} from './fixtures';
+
+describe( 'selectors', () => {
+	const initialState = {
+		productsList: reducer( undefined, { type: 'INIT' } ),
+	};
+
+	describe( '#getProductsList', () => {
+		it( 'should return the initial products list', () => (
+			expect( getProductsList( initialState ) ).to.eql( {} )
+		) );
+
+		it( 'should return a hydrated products list', () => {
+			const hydrated = {
+				productsList: reducer( initialState.productsList, {
```

Comment:
Could we please stub data? For a selector I'm mostly interested in the data contract. For anyone reading this test, they'd need to step through or console log to see what the structure looks like.